config/locales/en.yml

```diff
@@ -4,7 +4,10 @@ en:
       default: '%-d %B %Y'  # 3 January 2019
       compact: '%d/%m/%Y'   # 03/01/2019
       datetime: '%-d %B %Y %-l:%M%P'  # 3 January 2019 2:45pm
-
+    subjects:
+      applicant: 'your client'
+      applicant_and_partner: 'your client %{conjunction} their partner'
```

Comment:
Doesn't look like the conjunction is set anywhere in this pr, are you preparing to use it for a future pr?

Reply:
It is defaulted to 'or' in config/locales/en.rb and setting it to 'and' is required by a future change.